pkg/unidling/metrics/doc.go

```diff
@@ -0,0 +1,3 @@
+// Package prometheus contains code for unidling related metrics
+// into the prometheus registry running in the openshift master
+package prometheus
```

Comment:
Style and language nits:
- Package name should match the directory name, 
- Avoid the use of "master" as this is not inclusive. "Control plane" is the preferred term for the core components of Kubernetes and OpenShift.

```suggestion
// Package metrics contains code for unidling controller metrics
package metrics
```